coldfront/plugins/qumulo/management/commands/migrate_allocation_attributes.py

```diff
@@ -0,0 +1,40 @@
+from coldfront.core.allocation.models import (
+    AllocationAttributeType,
+    Allocation,
+    AllocationAttribute,
+)
+from coldfront.core.resource.models import Resource
+from django.core.management.base import BaseCommand
+
+from django.db.models import OuterRef, Subquery
```

Comment:
@cdeanj2, please refresh the working branch from the main branch since this code has already been added to the main branch.